src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -30,6 +31,7 @@ default boolean isSameStateAs(ReadOnlyPerson other) {
                 && other.getName().equals(this.getName()) // state checks here onwards
                 && other.getPhone().equals(this.getPhone())
                 && other.getEmail().equals(this.getEmail())
+//                && other.getAppointment().equals(this.getAppointment())
```

Comment:
Did you intend to comment this?  Note that irrelevant comments added by you should be removed in your submissions.